lib/background_worker.rb

```diff
@@ -1,14 +1,18 @@
-module BackgroundWorker
-  VERSION = '0.0.1'
-
-  require 'background_worker/base'
-  require 'background_worker/persistent_state'
+require 'background_worker/version'
+require 'background_worker/base'
+require 'background_worker/persistent_state'
 
+module BackgroundWorker
   def self.after_exception(&block)
     if block
       @@after_exception = block
-    else
-      @@after_exception if defined?(@@after_exception)
+      return
+    end
+
+    @@after_exception ||= lambda  do |e|
+      logger.error '** No after_exception handler installed **'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.